source/authentication/custom-jwt.txt

```diff
@@ -866,16 +866,16 @@ for your {+service-short+} app. You will learn how to:
 Prerequisites
 ~~~~~~~~~~~~~
 
-You can use any {+service-short+} app for this tutorial. If you do not have 
-a test app to use, you can :ref:`create an {+app+} <create-a-realm-app>` or 
+You can use any App Services app for this tutorial. If you do not have 
```

Comment:
```suggestion
You can use any App Services App for this tutorial. If you do not have 
```

can you do a check across the entire repo that we're doing capitalization correctly here? 

it should be 'App Services App'

Reply:
Corrected throughout